tests/conftest.py

```diff
@@ -15,3 +23,45 @@ def vcr_config():
     return {
         "filter_headers": [CROSSREF_HEADER_KEY, SEMANTIC_SCHOLAR_HEADER_KEY],
     }
+
+
+@pytest.fixture()
+def unique_tmp_subfolder(tmp_path):
+    yield tmp_path
+    # Cleanup after the test
+    if tmp_path.exists():
+        shutil.rmtree(tmp_path, ignore_errors=True)
+
+
+@pytest.fixture()
+def agent_module_dir(unique_tmp_subfolder, monkeypatch):
+    """Set up a unique temporary folder for the agent module."""
+    monkeypatch.setenv("PQA_HOME", str(unique_tmp_subfolder))
+    importlib.reload(paperqa.agents)
+    return unique_tmp_subfolder
+
+
+@pytest.fixture()
+def agent_index_dir(agent_module_dir):
+    return str(agent_module_dir / ".pqa/indexes")
+
+
+@pytest.fixture(name="agent_test_kit")
+def fixture_agent_test_kit() -> Answer:
+    return Answer(question="What is is a self-explanatory model?")
```

Comment:
I think I used the name "test kit" in the past because it gave a bunch of objects. This one is just an `Answer`, so can you name it like `stub_answer`?